Xamarin.Forms.Core/Items/ScrolledEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core.Items
+{
+	public class ScrolledEventArgs : EventArgs
+	{
+		public double HorizontalDelta { get; private set; }
```

Comment:
```suggestion
		public double HorizontalDelta { get; }
```

same for other auto-properties